.phpunit.cache/test-results

```diff
@@ -0,0 +1 @@
in\\ParserExceptionsTest::testBackgroundWithTag":0,"Tests\\Behat\\Gherkin\\ParserExceptionsTest::testEndlessPyString":0,"Tests\\Behat\\Gherkin\\ParserExceptionsTest::testWrongStepType":0,"Tests\\Behat\\Gherkin\\ParserExceptionsTest::testMultipleBackgrounds":0,"Tests\\Behat\\Gherkin\\ParserExceptionsTest::testMultipleFeatures":0,"Tests\\Behat\\Gherkin\\ParserExceptionsTest::testTableWithoutRightBorder":0,"Tests\\Behat\\Gherkin\\ParserTest::testParser#addition":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#background":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#background_title":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#big_pystring":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#clean_tags":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#commented_out":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#comments":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#complex_descriptions":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#empty":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#empty_scenario":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#empty_scenario_without_linefeed":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#empty_scenarios":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#fibonacci":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#hashes_in_quotes":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#issue_13":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#ja_addition":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#long_title_feature":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#multiline_name":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#multiline_name_with_newlines":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#multiplepystrings":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#outline_with_multiple_examples":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#outline_with_spaces":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#outline_with_step_table":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#pystring":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#ru_addition":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#ru_commented":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#ru_consecutive_calculations":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#ru_division":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#start_comments":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#tables":0.002,"Tests\\Behat\\Gherkin\\ParserTest::testParser#tags_sample":0.004,"Tests\\Behat\\Gherkin\\ParserTest::testParser#test_unit":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#trimpystring":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParser#undefined_multiline_args":0.001,"Tests\\Behat\\Gherkin\\ParserTest::testParserResetsTagsBetweenFeatures":0,"Tests\\Behat\\Gherkin\\ParserTest::testSingleCharacterStepSupport":0,"Tests\\Behat\\Gherkin\\ParserTest::testParsingManyCommentsShouldPass":0.001}}
```

Comment:
Should this have been committed? I think it should be gitignored?

Reply:
oops, nice catch 😆